sys/include/net/ng_netdev.h

```diff
@@ -145,13 +145,13 @@ typedef struct {
 
     /**
      * @brief   This function is called by a MAC layer when a message of type
-     *          NETDEV_MSG_EVENT_TYPE was received
+     *          @ref NG_NETDEV_MSG_TYPE_EVENT was received
      *
      * @param[in] dev           network device descriptor
-     * @param[in] event_type    event type, given by @ref msg_t::value in the
-     *                          received message
+     * @param[in] event_type    event type, given by msg_t::content::value
+     *                          in the received message
      */
-    void (*isr_event)(ng_netdev_t *dev, uint16_t event_type);
+    void (*isr_event)(ng_netdev_t *dev, uint32_t event_type);
```

Comment:
@haukepetersen was there a reason you chose `uint16_t` over `uint32_t`. Since the value comes from `msg_t.content.value` I thought it made sense to use the same type